automated install/basic-install.sh

```diff
@@ -1175,6 +1220,8 @@ installConfigs() {
 
         # Load final service
         systemctl daemon-reload
+    elif is_command openrc; then
+        install -T -m 0755 "${PI_HOLE_LOCAL_REPO}/advanced/Templates/pihole-FTL.openrc" "/etc/init.d/pihole-FTL"
```

Comment:
Just a niggle, but it's the kind of thing that makes my brain itchy... 😏 

This line is almost, excepting `service` vs `openrc`, exactly the same as the line in the catchall `else` block.

Maybe we could reduce the duplication here by moving the check for openrc into the `else` block and using that to set a variable in place of either `service` or `openrc` in the command.

Spitballing, and not neccaserily the right way to do it:

```bash
local INIT="service"
if is_command openrc: then
  INIT="openrc"
fi

install -T -m 0755 "${PI_HOLE_LOCAL_REPO}/advanced/Templates/pihole-FTL.${INIT}" '/etc/init.d/pihole-FTL'
```

I'm just being fussy - it's not a blocker